Makefile

```
@@ -159,11 +159,17 @@ kind-clean: ## Removes the kind instance if it exists.
 
 .PHONY: kind-run
 kind-run: export KUBECONFIG = $(KIND_KUBECONFIG)
-kind-run: kind-setup install run-operator ## Runs the operator on the local host but configured for the kind cluster
+kind-run: kind-setup kind-minio install run-operator ## Runs the operator on the local host but configured for the kind cluster
 
 kind-e2e-image: docker-build
 	$(e2e_make) kind-e2e-image
 
+.PHONY: kind-minio
+kind-minio: export KUBECONFIG = $(KIND_KUBECONFIG)
+kind-minio: kind-setup
+kind-minio:
+	kubectl apply -f $(SAMPLES_ROOT_DIR)/deployments/minio.yaml
+
```

I think it's safe to assume that the minio only really needs to be applied once. I propose to introduce a "sentinel file" that prevents subsequent `make kind-run` from applying the same manifests over again, thus speeding up local dev.
```suggestion
.PHONY: kind-minio
kind-minio: $(minio_sentinel)

$(minio_sentinel): export KUBECONFIG = $(KIND_KUBECONFIG)
$(minio_sentinel): kind-setup
	kubectl apply -f $(SAMPLES_ROOT_DIR)/deployments/minio.yaml
	@touch $@
```

---

Yeah I agree, the minio instance will probably not really change when developing. Thanks for the suggestion.